README.md

```diff
@@ -1,11 +1,454 @@
+===
+
+If the NativeScript CLI does not behave as expected, you might be facing a configuration issue. For example, a missing `JAVA` path. To check if your system is configured properly for the NativeScript CLI, run the following command.
+
+```bash
+tns doctor
+```
+
+This command prints warnings about current configuration issues and provides basic information about how to resolve them.
+
+If addressing the configuration issues does not resolve your problem, you can [report an issue](https://github.com/NativeScript/nativescript-cli/blob/master/CONTRIBUTING.md#report-an-issue) or [post in the NativeScript page in Google Groups](https://groups.google.com/forum/#!forum/nativescript).
+
+[Back to Top][1]
+
+Known Issues
+===
+
+* You cannot synchronize changes to apps on Android 4.3 devices and on some Samsung devices using the `run android` command.<br/>**Workaround:** Upgrade to a later version of Android to be able to use the `livesync` command. If you need to develop for Android 4.3 devices, re-build and re-deploy your app to get your latest changes on device.
+
+* On macOS systems with configured firewall or web proxy, when you run a command, the operation might not release the command line and you might not be able to run other commands until you break the current operation.<br/>If you have enabled feature usage tracking for the NativeScript CLI, but you have not authenticated with the firewall or web proxy on your macOS system, the NativeScript CLI might not release the command line after you run a command. To continue working with the NativeScript CLI, you need to break the current operation by pressing `Ctrl+C`.<br/>**Workaround:** Authenticate with the firewall or web proxy.
+    1. Close the terminal.
+    1. Run Safari.
+    1. Attempt to open a web page.
+    1. Provide your authentication credentials for accessing the Internet.
+    1. Run the terminal and continue working with the NativeScript CLI.
+
+[Back to Top][1]
+
+How to Contribute
+===
+
+To learn how to log a bug that you just discovered, click [here](https://github.com/NativeScript/nativescript-cli/blob/master/CONTRIBUTING.md#report-an-issue).
+
+To learn how to suggest a new feature or improvement, click [here](https://github.com/NativeScript/nativescript-cli/blob/master/CONTRIBUTING.md#request-a-feature).
+
+To learn how to contribute to the code base, click [here](https://github.com/NativeScript/nativescript-cli/blob/master/CONTRIBUTING.md#contribute-to-the-code-base).
+
+[Back to Top][1]
+How to Build
+===
+```
+git clone https://github.com/NativeScript/nativescript-cli
+cd nativescript-cli
+git submodule update --init
+npm install
+grunt
+```
+
+To use the locally built CLI instead `tns` you can call `PATH_TO_CLI_FOLDER/bin/tns`. For example:
+`PATH_TO_CLI_FOLDER/bin/tns run ios|android`
+
+> NOTE: You should have [SSH key set](https://github.com/settings/keys) for your Git in order add the submodule via [smart git protocol](https://github.com/settings/keys).
```

Comment:
we do not have this requirement anymore because the submodule is referenced via https instead of ssh